src/components/BottomNav/BottomNav.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useRef, useState } from 'react'
+import { useMatch } from 'react-router-dom'
+import { CSSTransition } from 'react-transition-group'
+
+import { navItems } from '@/config/nav'
+import { transitions } from '@/shared/theme'
+
+import { Container, NavLink, NavTitle } from './BottomNav.style'
+
+export const BottomNav = () => {
+  const [open, setOpen] = useState(false)
+  const match = useMatch
```

Comment:
What's the point of this?

Reply:
This is a workaround to use hook inside render

Reply:
I see now. However, I think this isn't a good idea. By "workaround" I guess you mean workaround for the eslint hooks rule. It's not there without a reason though. Rules of hooks say we cannot call hooks conditionally and that's what we're doing here by triggering inside a loop. You can create a wrapper for `NavLink` instead that will call the hook and manage it's `active` state autonomously